tests/simtel/test_simtelfile.py

```diff
@@ -237,3 +239,13 @@ def test_history_meta():
         assert isinstance(f.telescope_meta, dict)
         assert len(f.global_meta) == 11
         assert len(f.telescope_meta) == 19
+
+
+def test_type_2033():
+    with SimTelFile(pixel_monitoring_path) as f:
+        e = next(iter(f))
+        assert e['pixel_monitorings'].keys() == e['telescope_events'].keys()
+        assert e['pixel_monitorings'][1].keys() == {
```

Comment:
Same as above about the number of gains, though here I am not sure cause I cannot tell without digging a bit if the keys are filled in the parsing or somewhere else.

Reply:
Cameras with just a single gain shouldn't have the `fadc_amp_lg` key, but the test file I produced has only lst and mst nectarcam. I will produce a test file with paranal.